Dockerfile

```diff
@@ -5,6 +5,14 @@ FROM ruby:2.4.2
 RUN mkdir /app
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y wget
+
+# install dockerize
+ENV DOCKERIZE_VERSION v0.6.1
```

Comment:
Can we combine this with the next `RUN` line to avoid exposing this environment variable to the app?

Reply:
changed